addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -168,110 +168,109 @@
-                            <div class="col-12 col-md-8 offset-md-4 col-lg-9 offset-lg-3">
-                                <ul class="nav nav-tabs o_wslides_nav_tabs o_wslides_doc_nav_tabs flex-nowrap" role="tablist" id="profile_extra_info_tablist">
-                                    <li class="nav-item">
-                                        <a t-att-class="'nav-link %s' % ('active' if active_tab == 'home' else '')"
-                                            id="home-tab" data-toggle="pill" href="#home" role="tab" aria-controls="home"
-                                            t-att-aria-selected="'true' if active_tab == 'home' else 'false'">
-                                            <i class="fa fa-home"/>
-                                            Course
-                                        </a>
-                                    </li>
-                                    <li t-if="channel.allow_comment" class="nav-item o_wslides_course_header_nav_review_documentation">
-                                        <a t-att-class="'nav-link %s' % ('active' if active_tab == 'review' else '')"
-                                            id="review-tab" data-toggle="pill" href="#review" role="tab" aria-controls="review"
-                                            t-att-aria-selected="'true' if active_tab == 'review' else 'false'">
-                                            Reviews<t t-if="rating_count"> (<t t-esc="rating_count"/>)</t>
-                                        </a>
-                                    </li>
-                                </ul>
+                    <!-- Row with navbar and sidebar -->
+                    <div t-if="channel.channel_type == 'documentation' and channel.is_member" class="row">
+                        <!-- Sidebar -->
+                        <div class="col-12 col-md-4 col-lg-3 mt-3 mt-md-0">
+                            <t t-call="website_slides.course_sidebar"/>
+                        </div>
+                        <!-- navbar -->
+                        <div class="col-12 col-md-8 col-lg-9 mx-sm-3 ml-md-n3">
+                            <div t-att-class="'tab-pane fade %s' % ('show active' if active_tab == 'home' else '')" role="tabpanel" aria-labelledby="home-tab">
+                                <t t-call="website_slides.course_slides_cards_navbar"/>
                             </div>
                         </div>
                     </div>
 
-                    <div class="tab-content pb-5" id="courseMainTabContent">
-                        <div t-att-class="'tab-pane fade %s' % ('show active' if active_tab == 'home' else '')" id="home" role="tabpanel" aria-labelledby="home-tab">
-                            <t t-if="channel.channel_type == 'documentation'" t-call="website_slides.course_slides_cards"/>
+                    <!-- Row with content -->
+                    <div class="row">
+                        <!-- ========== SIDEBAR ========== -->
```

Comment:
and rename this to `<!-- ========== TRAINING SIDEBAR ========== -->`
unless its possible to factorise a bit both calls to sidebar.
I understand that for Documentation only, sidebar is in its own `div class="row"`, then you have another `div class="row"`in which you have the content. And for training, sidebar and content are in the same `div class="row"`.
Is it intended ?